sw/device/lib/base/testing/meson.build

```diff
@@ -12,6 +12,16 @@ sw_lib_testing_bitfield = declare_dependency(
   )
 )
 
+sw_lib_testing_hardened = declare_dependency(
```

Comment:
we probably need to update the Bazel configuration as well.

Reply:
Thanks! Just pushed an update.